websites/Y/YouTube Music/presence.ts

```diff
@@ -277,8 +283,21 @@ function updateSongTimestamps(useTimeLeft: boolean) {
 			.split(" / "),
 		[currTimes, totalTimes] = element;
 
-	mediaTimestamps = presence.getTimestamps(
-		useTimeLeft ? presence.timestampFromFormat(currTimes) : Date.now(),
-		useTimeLeft ? presence.timestampFromFormat(totalTimes) : null
-	);
+	if (useTimeLeft) {
+		mediaTimestamps = presence.getTimestamps(
+			presence.timestampFromFormat(currTimes),
+			presence.timestampFromFormat(totalTimes)
+		);
+	} else {
+		// leaving this here until it gets fixed in the extension, just in case
+		// mediaTimestamps = [
+		// 	Date.now() / 1000 - presence.timestampFromFormat(currTimes),
+		// 	null,
+		// ];
+		mediaTimestamps = presence.getTimestamps(
+			presence.timestampFromFormat(currTimes),
+			null
+		);
+		delete mediaTimestamps[1];
```

Comment:
All you're doing now is setting `Math.floor(Date.now() / 1000)` in a overcomplicated fashion.
The readable equivalent would be
```suggestion
		mediaTimestamps[0] = Math.floor(Date.now() / 1000);
```
However the startTimestamp is also already being set [here](https://github.com/PreMiD/Presences/blob/40d3e76322749e1bc94b20d68062ad4b85911f0b/websites/Y/YouTube%20Music/presence.ts#L170) of the code.
So you'll have to properly take the timestampFromFormat into account